code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -26,7 +26,7 @@
 	mutanteyes = /obj/item/organ/eyes/pod
 	mutantheart = /obj/item/organ/heart/pod
 	mutantliver = /obj/item/organ/liver/pod
-	mutantlungs = /obj/item/organ/lungs/pod
+	mutantlungs = /obj/item/organ/lungs/lavaland
```

Comment:
please make this modular or at least add the necessary edit comments as per our modularization guidelines